plugins/inventory/kubevirt.py

```diff
@@ -6,7 +6,7 @@
 __metaclass__ = type
 
 DOCUMENTATION = '''
-    name: kubevirt
+    name: community.kubevirt.kubevirt
```

Comment:
```suggestion
    name: kubevirt
```
This one needs to stay :)